easybuild/easyconfigs/c/CuPy/CuPy-11.4.0-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'PythonBundle'
+
+name = 'CuPy'
+version = '11.4.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://cupy.dev'
+description = "CuPy is an open-source array library accelerated with NVIDIA CUDA."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+builddependencies = [
+    ('pytest', '7.2.2'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('CUDA', '11.4.1', '', SYSTEM),
+    ('cuDNN', '8.2.2.26', versionsuffix, SYSTEM),
+    ('NCCL', '2.10.3', versionsuffix),
+    ('cuTENSOR', '1.6.1.5', versionsuffix, SYSTEM),
+    # Adding cuSPARSELt currently makes the build fail. Keeping it around for later versions to pick up.
+    # ('cuSPARSELt', '0.3.0.3', versionsuffix, SYSTEM),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_LOWER_SOURCE]}
+
+# A bunch of the tests are failing or are just having problems.
+_skip_tests = [
+    'tests/cupyx_tests/scipy_tests',
+    'tests/cupyx_tests/distributed_tests',
+    'tests/cupyx_tests/tools_tests',
+    'tests/example_tests',
+    'tests/cupy_tests/testing_tests/test_parameterized.py',
+    'tests/cupy_tests/core_tests/test_carray.py::TestCArray32BitBoundary_param_',
+    'tests/cupy_tests/fft_tests/test_fft.py',
+    # float16 has too low precision for these tests as they are written
+    # See https://github.com/easybuilders/easybuild-easyconfigs/pull/17526#issuecomment-1470843170 for details.
+    'tests/cupy_tests/linalg_tests/test_product.py::TestProduct',
```

Comment:
So, turns out this syntax isn't a thing in pytest. This line does nothing. Looks like this syntax https://github.com/pytest-dev/pytest/issues/3198 but this was never supported, the issue was closed in favor of a regex of what to include. 
Needs to be a different flag `-k "not test_tensordot"`

Ugh. This sucks. It won't even let us specify the full name, like "TestProduct::test_tensordot", just the last part.

Reply:
So options are
1. let this fail on A40, whoever installs it just has to ignore test failures.
2. patch this test out
3. add the -k flag, which just really sucks if we ever need to add more than one.

Regardless, this broken ::TestProduct ignore line should be removed, as it does nothing